### ISSUE TITLE
Remove Unnecessary Icon from Editor

### ISSUE THREAD

**Issue by shaneeshpatel1** [ailon/markerjs#63]
Hello @ailon,

We have set the default Marker - FreehandMarker by creating new marker, but there is an issue in editor display. We have already purchased the license.  We have to open by default Pen property open in editor thatswhy this is coming.

We have attached a screenshot for the reference. Please suggest how to remove this.
![Some Icon is coming in editor](https://user-images.githubusercontent.com/98814642/163832605-5f7d10e3-e62b-4cb1-ae0d-00a37d6b302d.png)

Regards,
Shaneesh Patel

**Comment by ailon** [comment]
This looks like controls of an already created marker. Must be that you start and finish creating marker from your code (for example by switching to select mode after you initiate creating a marker or something like that). Hard to say without a repro. Would it be possible for you to create a repro on codesandbox or similar?

You can send it to me privately at info@markerjs.com.

**Comment by shaneeshpatel1** [comment]
Hello @ailon 

I have sent the email to you. please check and revert.

Thanks
Shaneesh Patel

**Comment by ailon** [comment]
Seems to be some concurrency/timing issue. Still trying to get to the bottom of it, but here's a quick workaround for now. Just adding a minimal delay before creating that first marker helps:

```js
  markerArea.show();
  setTimeout(function () {
    markerArea.createNewMarker("FreehandMarker");
  }, 10);
```

**Comment by ailon** [comment]
This should be fixed now in v2.21.2. Thank you for reporting this bug!
